benchmarks/triton_kernels_benchmark/gemm_splitk_benchmark.py

```diff
@@ -177,7 +177,7 @@ def benchmark(M, N, K, provider):
         raise NotImplementedError(f'Unsupported provider {provider}')
 
     tflops = lambda mean: 2 * M * N * K * (1e-12) / (mean * 1e-3)
-    gbps = lambda mean: 2 * (M * K + K * N) + 4.0 * (M * N) * (1e-9) / (mean * 1e-3)
+    gbps = lambda mean: 2 * ((M * K + K * N) + 4.0 * (M * N)) * (1e-9) / (mean * 1e-3)
```

Comment:
Why? Should it be like:


```suggestion
    gbps = lambda mean: (2 * (M * K + K * N) + 4.0 * (M * N)) * (1e-9) / (mean * 1e-3)
```
?